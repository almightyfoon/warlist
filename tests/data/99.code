c3501b_-0x8D7p7p0d0d0d7s7s7shFhF8n8n___-0f6Y7p7u8tgcgc867-7-7X7W7X7W7X7Wi6