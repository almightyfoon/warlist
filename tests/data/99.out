https://conflictchamber.com/#c3501b_-0x8D7p7pkWkWkW7s7s7shFhF8n8n___-0f6Y7pkU8tgcgc867-7-7X7W7X7W7X7Wi6

Khador Army - 78 / 75 points
[Theme] Jaws of the Wolf

!!! This army must contain 75 points or fewer

[Harkevich 1] Kommander Harkevich, the Iron Wolf [+28]
 - Destroyer [14]
 - Destroyer [14]
 - Juggernaut [13]
 - Juggernaut [13]
 - Juggernaut [13]
 - Kodiak [13]
 - Kodiak [13]
 - Kodiak [13]
Greylord Forge Seer [0(4)]
Greylord Forge Seer [0(4)]
Manhunter [0(4)]
Manhunter [0(4)]

Khador Army - 73 / 75 points
[Theme] Winter Guard Kommand

[Sorscha 2] Forward Kommander Sorscha [+27]
 - Destroyer [14]
 - Spriggan [17]
Kovnik Jozef Grigorovich [4]
Winter Guard Artillery Kapitan [0(3)]
Winter Guard Artillery Kapitan [3]
Battle Mechaniks (min) [3]
Winter Guard Mortar Crew [0(5)]
Winter Guard Mortar Crew [0(5)]
Winter Guard Rifle Corps (min) [8]
 - Winter Guard Rocketeer (3) [6]
Winter Guard Rifle Corps (min) [8]
 - Winter Guard Rocketeer (3) [6]
Winter Guard Rifle Corps (min) [8]
 - Winter Guard Rocketeer (3) [6]
Winter Guard Gun Carriage [17]
